welcome/index.adoc

```diff
@@ -289,3 +327,14 @@ In addition to infrastructure metrics, you can also scrape and view metrics for
 - **xref:../support/remote_health_monitoring/about-remote-health-monitoring.adoc#about-remote-health-monitoring_about-remote-health-monitoring[Remote health monitoring]**: {product-title} collects anonymized aggregated information about your cluster and reports it to Red Hat via Telemetry and the Insights Operator. This information allows Red Hat to improve {product-title} and to react to issues that impact customers more quickly. You can view the xref:../support/remote_health_monitoring/showing-data-collected-by-remote-health-monitoring.adoc#showing-data-collected-by-remote-health-monitoring_showing-data-collected-by-remote-health-monitoring[data collected by remote health monitoring].
 ////
 endif::[]
+
+ifdef::openshift-rosa[]
+== Cluster administrator activities
+
+While cluster maintenance and host configuration is performed by the Red Hat Site Reliability Engineering (SRE) team, other ongoing tasks on your {product-title} {product-version} cluster can be performed by {product-title} cluster administrators. As a {product-title} cluster administrator, the ROSA documentation helps you:
+
+- *Manage Dedicated Administrators*: Grant or revoke permissions to `dedicated admin` users.
+- *Work with Logging*: Learn about {product-title} logging and configure the logging add-on services.
```

Comment:
Small nit...should this be `{product-title} Logging` (uppercase) because it is a product name? (See [Understanding Red Hat OpenShift Logging](https://docs.openshift.com/container-platform/4.7/logging/cluster-logging.html).)